src/custom/state/operator/actions.ts

```diff
@@ -0,0 +1,18 @@
+import { createAction } from '@reduxjs/toolkit'
+
+// modeled after state/user/actions
+export interface SerializedToken {
+  chainId: number
+  address: string
+  decimals: number
+  symbol?: string
+  name?: string
+}
+
+export interface Tip {
```

Comment:
nit: I think here you don't model a "tip", only a `TipQuote` (so it might be more precise) but since Tip is just a number, probably we don't need any type "Tip" for anything else, so I think could be fine to relax language and leave `Tip` here :)